tools/f18/CMakeLists.txt

```diff
@@ -71,13 +71,3 @@ add_custom_target(module_files ALL DEPENDS ${MODULE_FILES})
 
 install(TARGETS f18 f18-parse-demo DESTINATION bin)
 
-file(COPY flang.sh
-  DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/bin"
-  FILE_PERMISSIONS
-    OWNER_READ OWNER_WRITE OWNER_EXECUTE
-    GROUP_READ GROUP_EXECUTE
-    WORLD_READ WORLD_EXECUTE
-)
-file(RENAME "${CMAKE_CURRENT_BINARY_DIR}/bin/flang.sh" "${CMAKE_CURRENT_BINARY_DIR}/bin/flang")
-
-install(PROGRAMS flang.sh DESTINATION bin RENAME flang)
```

Comment:
Further to the comment about renaming it to flang-tmp, this should be reinstated.

Reply:
Done!